content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -6,6 +6,14 @@ weight: 100
 
 {{<glossary_definition term_id="node-pressure-eviction" length="short">}}</br>
 
+{{<note>}}
+The eviction signals, thresholds and metrics that were introduced as part of the
+_split image_ filesystem feature, which enables support for the `containerfs`
+filesystem, require the Kubernetes v{{< skew currentVersion >}} release, which
+includes alpha versions of the feature, to enable the `KubeletSeparateDiskGC`
```

Comment:
Maybe you can just mention that the feature needs to be enabled and drop mention of alpha.

Reply:
Done.